packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -176,7 +180,7 @@ export const DataGrid: React.FC<GridProps> = (props) => {
       setSelectionType,
       setAllowSelection,
       setPageSize,
-      setCurPage,
+      setPage,
```

Comment:
This is an API breaking change by renaming the function. You need to keep the key the same.